### PR TITLE
Add fixtures description to the development guide #1081

### PR DESCRIPTION
**Pull request by LightFelicis**
Closes #1084

### CODE REVIEW

**Review by AbhijithGanesh — CHANGES_REQUESTED** [review]
Please reference django docs too, that will make this complete.

**Review by AbhijithGanesh — APPROVED** [review]

**Comment by LightFelicis** [comment]
@AbhijithGanesh do you have merging rights? :-)

**Comment by AbhijithGanesh** [comment]
> 
> 
> @AbhijithGanesh do you have merging rights? :-)

Only @brylie  does, soon @gorkemarslan will get it.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -63,6 +63,27 @@ Certain resources, such as CSS and JavaScript files, need to be served from a st
...
+there are two [fixtures](https://docs.djangoproject.com/en/3.2/howto/initial-data/):
+
+* Sample threads, located in `project/data/sample_threads.json`
+* Sample categories, located in `project/data/categories.json`
+
+Run the following commands to load fixtures:
+
+```py
+python manage.py loaddata ./data/sample_threads.json
+python manage.py loaddata ./data/categories.json
```
Comment (gorkemarslan) [inline CONTRIBUTING.md:85]: To show an example from the `sample_threads.json` [file](https://github.com/CiviWiki/OpenCiviWiki/blob/develop/project/data/sample_threads.json), which will be updated soon:
```json
    {
        "model": "api.thread",
        "fields": {
            "title": "Liquid Fluoride Thorium Reactors",
            "summary": "...",
            "category_id": "2",
            "author_id": "2",
            "created": "2016-10-09 21:07:16.848433+00"
        }
    },
```
Notice that in order to load thread data, we use `category_id` field, so it is expected to be loaded `category` data already. Could you write the line to load `category` data just above the line for `thread` data?

**Review by gorkemarslan — CHANGES_REQUESTED** [review]

**Comment by gorkemarslan** [comment]
@LightFelicis, after issue #1079 is fixed, we'll be ready to approve your changes. It is good to see that the guide you prepared really works for the project. If not, we should make changes in the project. Then, we might want you to add a couple of lines here.

We're grateful for your contributions. :blush:

**Comment by AbhijithGanesh** [comment]
Yes totally, Either you can send in your changes and we can approve it for merging

**Comment by brylie** [comment]
@LightFelicis, thank you for your contribution :smiley: 

When creating a pull request that will resolve an issue, please [link the pull request to the related issue(s)](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) by using one of the [supported keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) followed by the issue number.

For example:

```
Closes #<issue-number>
```

## Further details

> You can link a pull request to an issue by using a supported keyword in the pull request's description or in a commit message.
> 
> - close
> - closes
> - closed
> - fix
> - fixes
> - fixed
> - resolve
> - resolves
> - resolved

**Comment by gorkemarslan** [comment]
@LightFelicis, could you look at [the requested change](https://github.com/CiviWiki/OpenCiviWiki/pull/1084#discussion_r721061520)?

**Comment by gorkemarslan** [comment]
@LightFelicis, there is a misunderstanding here. I don't want you to add example fixtures. I just want to explain the change I requested clearly. :smile:  So remove the example.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -63,6 +63,52 @@ Certain resources, such as CSS and JavaScript files, need to be served from a st
...
+```json
+    {
+        "fields": {
+            "name": "Agriculture"
+        },
+        "model": "categories.Category",
+        "pk": 0
+    }
+```
+
```
Comment (gorkemarslan) [inline CONTRIBUTING.md]: You can remove them as I explained.

**Review by gorkemarslan — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -63,6 +63,52 @@ Certain resources, such as CSS and JavaScript files, need to be served from a st
...
+```json
+    {
+        "fields": {
+            "name": "Agriculture"
+        },
+        "model": "categories.Category",
+        "pk": 0
+    }
+```
```
Comment (brylie) [inline CONTRIBUTING.md]:
```suggestion
  Example of values:

```json
```
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -63,6 +63,52 @@ Certain resources, such as CSS and JavaScript files, need to be served from a st
...
+        "fields": {
+            "title": "Liquid Fluoride Thorium Reactors",
+            "summary": "...",
+            "category_id": "2",
+            "author_id": "2",
+            "created": "2016-10-09 21:07:16.848433+00"
+        }
+    }
+```
+
```
Comment (brylie) [inline CONTRIBUTING.md]: Remove this code as follows:

```suggestion
```

**Review by brylie — CHANGES_REQUESTED** [review]

**Comment by gorkemarslan** [comment]
@LightFelicis, do you need any further clarification or help for the requested changes?

**Comment by AbhijithGanesh** [comment]
@brylie can you please check this?

**Comment by AbhijithGanesh** [comment]
I think this PR should be merged once Linting is taken care of, otherwise it won't pass tests

**Review by gorkemarslan — APPROVED** [review]

**Comment by brylie** [comment]
@all-contributors please add @LightFelicis for docs

**Pull request merged**